src/tests/validity/mv.ab

```diff
@@ -0,0 +1,12 @@
+import * from "std/fs"
+main {
+    let tmpdir = unsafe $mktemp -d /tmp/amber-XXXX$
```

Comment:
If mktemp fails it will be an empty variable resulting in `rm -rf` being called without argument.
It seems like that won't remove anything so this might be a nitpick. I still feel like it's a somewhat dangerous gap to leave around regardless.
Maybe just don't use rm -rf here, the number of files is very limited

Reply:
this approach is used in multiple [stdlib](https://github.com/amber-lang/amber/tree/master/src/tests/stdlib) test file like [file_append.ab](https://github.com/amber-lang/amber/blob/master/src/tests/stdlib/file_append.ab#L3) and [get_env_var.ab](https://github.com/amber-lang/amber/blob/master/src/tests/stdlib/get_env_var.ab#L4) maybe we should address this concern in different pr to migrate all these tests to the appropriate way?

Reply:
Yes it is the case to address this issue in a dedicated issue.
We already discussed few days ago about something that is executed automatically to all the tests in this way this code is not needed and they are sandbox.

Reply:
@karpfediem can you create an issue for this? Thanks for the feedback!

Reply:
@Ph0enixKM opened https://github.com/amber-lang/amber/issues/353